### ISSUE TITLE
UserAgent logs too much content

### ISSUE THREAD

**Issue by mantas** [zammad/zammad#4947]
### Used Zammad Version

6.2

### Environment

- Installation method: any
- Operating system (if you're unsure: `cat /etc/os-release` ): any
- Database + version: any
- Elasticsearch version: any
- Browser + version: any

### Actual behaviour

`UserAgent` logs all responses for some types of requests. Including content of successful responses. Which may generate a lot of noise and take up resources. However, in some cases it may be useful for debugging.

### Expected behaviour

To be discussed. For example, `UserAgent` shall log errors only.

### Steps to reproduce the behaviour

Use one of the following features:

- External data source
- Idoit integration
- Github/Gitlab integration
- SAML
- Webhook

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by mantas** [comment]
@dk @mgruner do you have anything to add?

**Comment by dk** [comment]
Nah I'm good thank you ... wrong dk most probably, though 🤣

**Comment by mantas** [comment]
> Nah I'm good thank you ... wrong dk most probably, though 🤣

Oh crap… Sorry dude. Thank you for pointing it out 😁

**Comment by mgruner** [comment]
@mantas AFAIR `Expected Behaviour` must first be specified. We discussed logging errors by default, but also the possibility for an admin to turn on logging (globally?) for all requests for debugging purposes, e.g. via ENV variable. This should be clarified by product first.

**Comment by mantas** [comment]
@mgruner Yes. That's why I added `specification required` label. Edited to be more clear.

ping the real @dominikklein :)
